code/Common/Exporter.cpp

```diff
@@ -370,7 +370,7 @@ aiReturn Exporter::Export( const aiScene* pScene, const char* pFormatId, const c
     // format. They will likely not be aware that there is a flag in the scene to indicate
     // this, however. To avoid surprises and bug reports, we check for duplicates in
     // meshes upfront.
-    const bool is_verbose_format = !(pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || MakeVerboseFormatProcess::IsVerboseFormat(pScene);
+    const bool is_verbose_format = !((pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || !MakeVerboseFormatProcess::IsVerboseFormat(pScene));
```

Comment:
Double negation like this does not look right. Are you sure it should not be 
```
(pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || MakeVerboseFormatProcess::IsVerboseFormat(pScene)
```

Reply:
no, because then if the scene has the flag (the scene is not verbose) it will set is_verbose_format to true, the idea was to make the line better fool-proof and remove the possibility of a fatal exception.
I just read the comment above and was not sure that was the intent in the line.

maybe instead of double negative:
```
MakeVerboseFormatProcess::IsVerboseFormat(pScene) || !(pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT)
```

This will result in the Verbose check always being run, and the flag as an extra step if the check results in negative.